Readme.md

```diff
@@ -7,3 +7,5 @@ To run the example use ``mvn exec:java``:
     $ mvn compile exec:java -Dhost=YOUR_CLUSTER_ID.REGION.aws.found.io -Dshield.user="username:password"
 
 Replace `YOUR_CLUSTER_ID` with your cluster id and `REGION` with the region the cluster is started in.
+
+_(Note: by default, if IPv4 and IPv6 are both enabled then it is undetermined which will be used. The system properties `-Dipv4=true|false` and `-Dipv6=true|false` can be used to turn them on/off explicitly. If they are enabled then the network/host should support routing to the Cloud endpoint, eg this is not the case within Docker containers by default)._
```

Comment:
Well, depending on the routing tables, it may take a while for the client to figure out if an ip6 address is usable or not. If the ELBs have many ip6 adresses, it will accumulate. How long did you wait? The initial reason for adding this address type awareness to the code was that customers without ipv6 support were seeing noisy exceptions in logs, not that they were unable to connect at all. Since then the ELB's may have been scaled up quite a bit.